### ISSUE TITLE
onPaste not firing

### ISSUE THREAD

**Issue by knubie** [ianstormtaylor/slate#3668]
#### What's the current behavior?

```jsx
<Editable onPaste={() => console.log("paste")} />
```
The handler is never fired.

Slate: 0.58.0
Browser: Chrome / Safari / Firefox / Edge
OS: Mac / Windows / Linux / iOS / Android

#### What's the expected behavior?
The handler should fire.

Based on my testing it seems that the call to the handler is being short-circuited in this conditional:
https://github.com/ianstormtaylor/slate/blob/master/packages/slate-react/src/components/editable.tsx#L926-L932

In other words, if one of the other predicates in the conditional is false, the handler never gets called.

In an unrelated note I think that having a predicate cause side effects is bad practice, and would recommend refactoring the way these handler work in that regard.

**Comment by mufasa71** [comment]
I am not sure but based on example given you need to wrap it with function

```
<Editable onPaste={() => console.log("paste")} />
```

**Comment by knubie** [comment]
@mufasa71 Just a mistake when writing the issue.
